src/sparseml/modifiers/obcq/base.py

```diff
@@ -58,8 +60,9 @@ class SparseGPTModifier(Modifier):
     quantize: Union[bool, Dict]
     dampening_frac: Optional[float] = 0.01
     sequential_update: Optional[bool] = True
-    prunen: Optional[int] = 0
-    prunem: Optional[int] = 0
+    mask_structure: str = "0:0"
+    prunen_: Optional[int] = None
+    prunem_: Optional[int] = None
```

Comment:
why keep both of these interfaces around if `mask_structure` must be defined? it seems currently it will clobber whatever you put in prunen or prunem? i'm questioning the justification for this change because of this complexity

Reply:
Both interfaces are not around anymore, infact specifying `prunen` or `prunem` will fail; they have been converted to private variables `prunen_` and `prunem_` ; Also removed from the docstring; are you saying it can lead to confusion? if yes what are your thoughts, should we rename them somehow? or anything else you would like to see?